packages/material-ui/src/Tab/Tab.test.js

```diff
@@ -37,6 +37,26 @@ describe('<Tab />', () => {
     skip: ['componentProp'],
   }));
 
+  it('should have a ripple by default', () => {
+    const wrapper = shallow(<Tab />);
```

Comment:
We are moving away from shallow tests. Please use the mount API.

Reply:
See https://github.com/mui-org/material-ui/pull/15864#pullrequestreview-242029949